Tabloid/client/src/components/users/UserProfileItem.js

```diff
@@ -12,9 +51,39 @@ export const UserProfileItem = ({ user }) => {
             <td>
                 {user.displayName}
             </td>
+            
             <td>
                 {user.userType.name}
             </td>
+            <td>{
+                user.isActive
+                    ?<CardLink href={"javascript:void(0)"} onClick={toggleConfirmDeactivate}>
+                    Deactivate User
+                    </CardLink>
+                    :<CardLink href={"javascript:void(0)"} onClick={toggleConfirmActivate}>
+                    Activate User
+                    </CardLink>
+                }
+            </td>
+            {
+                confirmDeactivate ?
+                <div>
+                <h5>Are you sure you want to Deactivate this User?</h5>
+                <button onClick={handleDeactivate} className="btn btn-primary mt-2 mr-5">Deactivate</button>
+                <a href="/users" className="btn btn-primary mt-2 mr-5">Go Back</a>
```

Comment:
Instead of using a link here, you could make it a button tag and add `onClick={toggleConfirmDeactivate}`. This would prevent a re-render of the page and having to communicate with the database again. Same for on line 83 with activate confirmation go back.

Reply:
Great job, thank you!